CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLWorker.cc

```diff
@@ -250,6 +251,20 @@ SiPixelLorentzAnglePCLWorker::SiPixelLorentzAnglePCLWorker(const edm::ParameterS
   min_drift_ = -1000.;  //-200.;(iConfig.getParameter<double>("residualMax"))
   max_drift_ = 1000.;   //400.;
 
+  for (int lay = 0; lay < 4; ++lay) {
+    for (int lad = 0; lad < 64; ++lad) {
+      for (int mod = 0; mod < 8; ++mod) {
+        newmodule[lay][lad][mod] = false;
+      }
+    }
+  }
+  newmodule[0][3][6] = true;
```

Comment:
I was hoping this could be configurable, i.e. pass a list of `DetId`s in the configuration and construct the list from those.

Reply:
Yes, I'd second that idea!

Reply:
@mmusich You mean like

```
          layer_ = tTopo->pxbLayer(detIdObj);
          ladder_ = tTopo->pxbLadder(detIdObj);
          module_ = tTopo->pxbModule(detIdObj);
            
          bool newmodule = false;
          if (layer_ == 1){
              if (ladder_ == 4 && module_ == 7) newmodule = true;
              if (ladder_ == 4 && module_ == 4) newmodule = true;
              if (ladder_ == 4 && module_ == 2) newmodule = true;
              if (ladder_ == 6 && module_ == 7) newmodule = true;
              if (ladder_ == 6 && module_ == 4) newmodule = true;
              if (ladder_ == 8 && module_ == 4) newmodule = true;
            }
```
?

Reply:
@wweiphy, I mean passing the list of special detids from configuration file and create a new group from them.